backend/cfg/cfg.ml

```diff
@@ -299,6 +301,8 @@ let dump_terminator ?(sep = "\n") ppf (terminator : terminator) =
   | Raise _ -> fprintf ppf "Raise%s" sep
   | Tailcall (Self _) -> fprintf ppf "Tailcall self%s" sep
   | Tailcall (Func _) -> fprintf ppf "Tailcall%s" sep
+  | Poll_and_jump return_label ->
+    Format.fprintf ppf "@[(poll_and_jump %a)@]" Label.print return_label
```

Comment:
(Inconsistent with the format used for other terminators.)

Reply:
I've now added `sep`, but that is all so far.